src/pages/TeamOverview/__tests__/TeamOverview.test.tsx

```diff
@@ -29,24 +31,24 @@ describe('TeamOverview', () => {
     });
 
     it('should render team overview users', async () => {
-        const teamOverview = {
+        const teamOverview: TeamOverviewType = {
             id: '1',
             teamLeadId: '2',
             teamMemberIds: ['3', '4', '5'],
         };
-        const userData = {
+        const userData: UserDataType = {
             id: '2',
             firstName: 'userData',
             lastName: 'userData',
             displayName: 'userData',
             location: '',
             avatar: '',
         };
-        jest.spyOn(API, 'getTeamOverview').mockImplementationOnce(() => Promise.resolve({} as any));
-        jest.spyOn(API, 'getUserData').mockImplementationOnce(() => Promise.resolve({} as any));
+        jest.spyOn(API, 'getTeamOverview').mockImplementationOnce(() => Promise.resolve(teamOverview));
```

Comment:
The `getTeamOverview` method, when mocked, should return the `teamOverview` data, with `TeamOverviewType`